app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -49,7 +49,7 @@ class PatientComponent extends Component {
 
   searchDemographics(event) {
     event.preventDefault();
-    const { gender, minAge, maxAge, startDate, endDate } = this.state;
+    const { gender, minAge, maxAge, startDate, endDate, livingStatus } = this.state;
```

Comment:
Merging for today. But next time, it would be better to have a separate pull request for removing the drug order tab. 😊